services/settings-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SettingsServiceDAL.java

```diff
@@ -308,6 +308,10 @@ public Setting updateSetting(Setting setting) {
         return fromParameterStore(updated);
     }
 
+    public void deleteSetting(Setting setting) {
+        deleteParameter(toParameterStore(setting));
+    }
```

Comment:
Why are these functions added, for future convenience? They don't appear to be used.

Reply:
They were added as I started working on the functionality and I switched gears to using the batch delete method from the SDK instead. I can see future functionality coming from them, but you are correct, they are not used today.